### PR TITLE
CRD3 dataset card

### PR DESCRIPTION
**Pull request by wilsonyhlee**
This PR adds additional information to the CRD3 dataset card.

### CODE REVIEW

File: datasets/crd3/README.md [hunk 1]
```diff
@@ -154,11 +174,11 @@ The data fields are the same among all splits.
 
 ### Dataset Curators
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+CRTranscript provided transcripts of the show; contributors of the Critical Role Wiki provided the abstractive summaries.
 
 ### Licensing Information
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+This work is licensed under a [Creative Commons Attribution-ShareAlike 4.0 International License][cc-by-sa]., as corresponding to the Critical Role Wiki https://criticalrole.fandom.com/
```
Comment (lhoestq) [inline datasets/crd3/README.md:181]: In the tags it is written cc-by-4.0, while here it is written cc-by-sa. Which one is correct ?

File: datasets/crd3/README.md [hunk 2]
```diff
@@ -104,37 +122,39 @@ The data fields are the same among all splits.
...
 |-------|------:|---------:|------:|
-|default|2942362|   2942362|2942362|
+|default|26,232|   3,470|4,541|
 
 ## Dataset Creation
 
 ### Curation Rationale
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+Dialogue understanding and abstractive summarization remain both important and challenging problems for computational linguistics. Current paradigms in summarization modeling have specific failures in capturing semantics and pragmatics, content selection, rewriting, and evaluation in the domain of long, story-telling dialogue.  CRD3 offers a linguistically rich dataset to explore these domains.
```
Comment (lhoestq) [inline datasets/crd3/README.md:131]: removed one space
```suggestion
Dialogue understanding and abstractive summarization remain both important and challenging problems for computational linguistics. Current paradigms in summarization modeling have specific failures in capturing semantics and pragmatics, content selection, rewriting, and evaluation in the domain of long, story-telling dialogue. CRD3 offers a linguistically rich dataset to explore these domains.
```

**Review by lhoestq — COMMENTED** [review]
Thanks for completing the dataset card :)

I just have one question about the license (see my comments)

File: datasets/crd3/README.md (hunk 1 above)
Comment (wilsonyhlee) [reply datasets/crd3/README.md]: Thanks for catching that! `cc-by-sa-4.0` is the correct one. I've updated accordingly.

**Review by lhoestq — APPROVED** [review]
Thanks ! looks all good :)

**Pull request merged**